### PR TITLE
Fix settings typos

### LINKED ISSUE #9: Preset Issue
The Display 2x2  presets has a bug and is Solo not 2x2 in the Config

### PR DESCRIPTION
**Pull request by peternewman**
Behind #12 or just merge this to get both...

Closes #9

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -106,53 +109,51 @@ class instance extends instance_skel {
...
+				break
 			case 'set_tally':
-				cmd = "CONFIGURATION:\n"+"Display SDI tally: "+opt.setting+"\n\n";
-				break;
+				cmd = 'CONFIGURATION:\n' + 'Display SDI tally: ' + opt.setting + '\n\n'
+				break
 			case 'set_widescreen_sd':
-				cmd = "CONFIGURATION:\n"+"Widescreen SD enable: "+opt.setting+"\n\n";
-				break;
+				cmd = 'CONFIGURATION:\n' + 'Widescreen SD enabled: ' + opt.setting + '\n\n'
```
Comment (peternewman) [inline index.js]: This fixes it for me.

Obviously I can't tell whether they've changed it over the years (like the output format), or it wasn't tested sufficiently initially (like the solo mode). We could add in some other stuff to send and receive both variants, but maybe that's excessive?

File: index.js [hunk 2]
```diff
@@ -472,49 +458,48 @@ class instance extends instance_skel {
...
-					this.checkFeedbacks('solo_enabled');
-					break;
-				case 'Widescreen SD enable':
-					this.getConfig().widescreenSD = value;
-					this.checkFeedbacks('widescreen_sd');
-					break;
+					this.getConfig().soloEnabled = value
+					this.checkFeedbacks('solo_enabled')
+					break
+				case 'Widescreen SD enabled':
```
Comment (peternewman) [inline index.js:496]: The matching change is here...

**Review by peternewman — COMMENTED** [review]
Just explaining one change...

File: index.js (hunk 1 above)
Comment (peternewman) [reply index.js]: It's not been fixed/still says enable all over the place in the Multiview16 too...

Comment (peternewman) [reply index.js]: The manual says enable for version 2.3 of the protocol, I'm on 2.8...

Comment (peternewman) [reply index.js]: The code now handles both variants just in case...

**Comment by peternewman** [comment]
@PerRoine @krocheck @josephdadams is someone able to take a look at this please? I think it fixes all the current issues with the module (aside from the output format settings already covered in #6 ).

**Comment by josephdadams** [comment]
@peternewman im adding you as a maintainer to this repo. Please clean up any issues

**Comment by peternewman** [comment]
Thanks @josephdadams . I think all the resolved ones are shut now.

Do you need to cut a release for this to end up in future Companion releases?

Although it would be good to get #6 in first as I think that completes the bulk of outstanding requests. I left a suggestion which I'd test run myself before of just having both variants available for backwards compatibility.

**Comment by josephdadams** [comment]
Is #6 ready to merge?

**Comment by peternewman** [comment]
Yeah I've just gone ahead and merged it now, so it's just the release that needs cutting thanks @josephdadams .

**Comment by josephdadams** [comment]
I cannot cut a release until the module is converted to companion v3 API. Looks like you are working on that in #14?

**Comment by peternewman** [comment]
Is that just a policy thing or a more recent change @josephdadams ? It looks like there have been some previous updates to old modules e.g. in https://github.com/bitfocus/companion/commit/08a29ec81f55a02b093a00a864ce8049d0c58007 ?

Yeah I'd started on that but may not have much time for a bit, again needing to work out which of the solutions discussed in #14 to do.

**Comment by josephdadams** [comment]
Policy. Just duplicate whatever code is needed and have separate modules for now. We can always merge later if needed. It’s not like there is a limit on module count.

**Pull request merged**